src/teleop/gui/src/store/modules/erd.js

```diff
@@ -0,0 +1,64 @@
+// initial state
+const state = {
+    route: [],
+    waypointList: [],
+    highlightedWaypoint: -1,
+    autonEnabled: false,
+    teleopEnabled: true,
```

Comment:
Are we sure we need all these? Seems like route, auton and teleop enabled maybe aren't super necessary but not sure if I missed them